portal/server/api.js

```diff
@@ -856,6 +856,28 @@ const getPreviousUtilisationReportsByBank = async (token, bankId) => {
   }
 };
 
+const getDueReportsByBank = async (token, bankId) => {
+  try {
+    if (!isValidBankId(bankId)) {
+      console.error('Getting due utilisation reports failed for id %s', bankId);
+      return false;
+    }
+    const response = await axios({
+      method: 'get',
+      url: `${PORTAL_API_URL}/v1/due-reports/${bankId}`,
```

Comment:
It's going to be worth doing it now so we can refactor this to `/v1/banks/:bankId/due-reports` please?